apps/agora/api/src/models/teams.ts

```diff
@@ -0,0 +1,34 @@
+// -------------------------------------------------------------------------- //
+// External
+// -------------------------------------------------------------------------- //
+import { Schema, model } from 'mongoose';
+
+// -------------------------------------------------------------------------- //
+// Internal
+// -------------------------------------------------------------------------- //
+import { Team, TeamMember } from '@sagebionetworks/agora/api-client-angular';
+
+// -------------------------------------------------------------------------- //
+// Schemas
+// -------------------------------------------------------------------------- //
+const TeamMemberSchema = new Schema<TeamMember>({
+  name: { type: String, required: true },
+  isprimaryinvestigator: { type: Boolean, required: true },
```

Comment:
Use camel case for variables.

```suggestion
  isPrimaryInvestigator: { type: Boolean, required: true },
```